scss/partials/_footer.scss

```diff
@@ -0,0 +1,138 @@
+footer {
+  @include breakpoint(desktop) {
+    width: 1440px;
+  }
+}
+
+.grid-container {
+  display: grid;
+}
```

Comment:
Either remove this and apply display to the root BEM element OR turn it into a _util class that gets extended.